portal-frontend/src/app/features/create-application-dialog/create-application-dialog.component.ts

```diff
@@ -79,11 +79,35 @@ export class CreateApplicationDialogComponent implements OnInit, AfterViewChecke
 
   onReadMoreClicked() {
     this.readMoreClicked = !this.readMoreClicked;
```

Comment:
These 4 all look the same to me, can you DRY it up?

Reply:
Updated!